pygpu/tests/main.py

```diff
@@ -1,8 +1,9 @@
 import os
+import nose.plugins.builtin
```

Comment:
Add a comment to why it need to be imported while we don't call:
nose.plugins.builtin

Why jenkins didn't broke?